download.sh

```diff
@@ -0,0 +1,16 @@
+USERNAME="admin"
+PASSWORD="mypassword"
+TIME=60 # seconds
+IP="192.168.1.197"
+PORT=554
+
+ffmpeg -i rtsp://$USERNAME:$PASSWORD@$IP:$PORT/h264Preview_01_main \
```

Comment:
does the script download into the current directory or can i set that? maybe it would make sense to add a flag to the download script for the directory and fallback to a "recordings" directory where the script is located.